### PR TITLE
Add references to orchestrator and workflow installations

### PR DESCRIPTION
**Pull request by masayag**
The PR splits the orchestrator installation section to install directly from sources or the helm repository.
For the workflows, the instructions are split between Helm and Kustomize.

Selection_144

### CODE REVIEW

File: content/docs/installation/_index.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ weight: 1
 The orchestrator solution is made of various independent components where each has its own
 installation method. At the moment we support a specialized helm chart to install everything
 from scratch on OpenShift or on Kubernetes. The installation is modular so if any of the pieces
-is already installed it can be made optional (more in the helm chart README.md below.
+is already installed it can be made optional (more in the helm chart README.md below).
```
Comment (gciavarrini) [inline content/docs/installation/_index.md:10]: Not sure if mentioning `README.md` can confuse the user. From the website, the user hasn't the perception of the README files.
Not a strong opinion on this, but I'd like to highlight this :)

File: content/docs/installation/orchestrator/orchestrator-helm-repository.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: Deploy from repository
+date: "2024-03-12"
+---
+
+{{< remoteMD "https://github.com/parodos-dev/orchestrator-helm-chart/blob/gh-pages/README.md?raw=true" >}}
```
Comment (gciavarrini) [inline content/docs/installation/orchestrator/orchestrator-helm-repository.md:6]: Not sure if related to this PR, but once rendered this page has a link to workflow installation
image

If i click this I'm redirected to https://www.parodos.dev/serverless-workflows-helm/ and i get: 
image
while if i click from the link available in the installation page, I'm redirect to https://www.parodos.dev/docs/installation/serverless-workflows-helm/ and I get: 
image

Is it ok?

File: content/docs/installation/workflows/serverless-workflows-helm copy.md
Comment (gciavarrini) [inline content/docs/installation/workflows/serverless-workflows-helm copy.md:1]: Why renaming as copy ( `<filename> copy.md` )?

File: content/docs/installation/orchestrator/orchestrator-helm-repository.md (hunk 2 above)
Comment (gabriel-farache) [reply content/docs/installation/orchestrator/orchestrator-helm-repository.md:6]: IMO it's not related to this PR, it's just that our doc is starting to be a mess and we need to harmonize/clean it in all the places we have docs (so the gh-pages of the helm chart repos and this repo for instance)

Comment (masayag) [reply content/docs/installation/orchestrator/orchestrator-helm-repository.md:6]: sadly yes, this page needs to be fixed on the origin repository. but the link is correct.

File: content/docs/installation/_index.md (hunk 1 above)
Comment (masayag) [reply content/docs/installation/_index.md:10]: tried to address this. WDYT?

File: content/docs/installation/orchestrator/orchestrator-helm-repository.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+title: Deploy from repository
```
Comment (gabriel-farache) [inline content/docs/installation/orchestrator/orchestrator-helm-repository.md:2]: from helm repository, right?

**Review by gabriel-farache — APPROVED** [review]
maybe one little thing that can be confusing bu OK overall

File: content/docs/installation/orchestrator/orchestrator-helm-repository.md (hunk 2 above)
Comment (masayag) [reply content/docs/installation/orchestrator/orchestrator-helm-repository.md:6]: > sadly yes, this page needs to be fixed on the origin repository. but the link is correct.

I'll correct my answer and point directly to the gh-pages which contains the update documentation:
https://github.com/parodos-dev/serverless-workflows-helm/blob/gh-pages/README.md

File: content/docs/installation/orchestrator/orchestrator-helm-repository.md (hunk 3 above)
Comment (masayag) [reply content/docs/installation/orchestrator/orchestrator-helm-repository.md:2]: yes, was trying to save few characters.

Comment (gabriel-farache) [reply content/docs/installation/orchestrator/orchestrator-helm-repository.md]: at the expense of making thing confusing for dumb people like me x))

File: content/docs/installation/_index.md (hunk 1 above)
Comment (gciavarrini) [reply content/docs/installation/_index.md]: Great! thanks

File: content/docs/installation/orchestrator/_index.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Orchestrator"
+date: 2024-03-03
+---
+
+The orchestrator solution is made of various independent components where each has its own
+installation method. At the moment we support a specialized helm chart to install everything
+from scratch on OpenShift or on Kubernetes. The installation is modular so if any of the pieces
+is already installed it can be made optional (more in the helm chart README.md below).
```
[the remark this answers is not in the document]
Comment (gciavarrini) [reply content/docs/installation/orchestrator/_index.md]: I think now it's very clear. Thank you!

**Review by gciavarrini — APPROVED** [review]
/lgtm

**Pull request merged**